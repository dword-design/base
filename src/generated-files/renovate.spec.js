import proxyquire from '@dword-design/proxyquire'

export default {
  base() {
    const self = proxyquire('./renovate', {
      '../package-config': { name: '@dword-design/base' },
    })
    expect(self).toMatchSnapshot(this)
  },
  'not base': function () {
    const self = proxyquire('./renovate', {
      '../package-config': { name: 'foo' },
    })
    expect(self).toMatchSnapshot(this)
  },
}
