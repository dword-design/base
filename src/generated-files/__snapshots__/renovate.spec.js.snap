// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renovate base 1`] = `
Object {
  "extends": Array [
    ":semanticCommits",
    ":semanticPrefixFix",
  ],
  "labels": Array [
    "maintenance",
  ],
  "lockFileMaintenance": Object {
    "enabled": true,
    "semanticCommitType": "chore",
  },
  "semanticCommitScope": null,
}
`;

exports[`renovate lock file commit type 1`] = `
Object {
  "extends": Array [
    ":semanticCommits",
  ],
  "ignorePaths": Array [
    ".github/workflows/build.yml",
  ],
  "labels": Array [
    "maintenance",
  ],
  "lockFileMaintenance": Object {
    "enabled": true,
  },
  "packageRules": Array [
    Object {
      "depTypeList": Array [
        "dependencies",
        "devDependencies",
        "peerDependencies",
        "optionalDependencies",
      ],
      "semanticCommitType": "fix",
    },
  ],
  "semanticCommitScope": null,
}
`;

exports[`renovate lock file fix commit type 1`] = `
Object {
  "extends": Array [
    ":semanticCommits",
    ":semanticPrefixFix",
  ],
  "ignorePaths": Array [
    ".github/workflows/build.yml",
  ],
  "labels": Array [
    "maintenance",
  ],
  "lockFileMaintenance": Object {
    "enabled": true,
  },
  "semanticCommitScope": null,
}
`;

exports[`renovate not base 1`] = `
Object {
  "extends": Array [
    ":semanticCommits",
    ":semanticPrefixFix",
  ],
  "ignorePaths": Array [
    ".github/workflows/build.yml",
  ],
  "labels": Array [
    "maintenance",
  ],
  "lockFileMaintenance": Object {
    "enabled": true,
    "semanticCommitType": "chore",
  },
  "semanticCommitScope": null,
}
`;
